module/metrics/execution.go

```diff
@@ -405,6 +406,14 @@ func NewExecutionCollector(tracer module.Tracer) *ExecutionCollector {
 		Buckets:   []float64{50, 100, 500, 1000, 5000, 10000},
 	})
 
+	transactionNormalizedTimePerComputation := promauto.NewHistogram(prometheus.HistogramOpts{
```

Comment:
can you add comments about how to read this metrics?

Say the `EstimatedComputationPerMillisecond` is `50`, then a `transaction_computation_per_ms` more than `1` means the EN took less time to finish executing this tx?

Reply:
I added a comment